### ISSUE TITLE
Bug in the LPIPS metric computation?

### ISSUE THREAD

**Issue by pschroeppel** [YuLiu-LY/SlotLifter#1]
Hi,

I think I discovered a bug in the LPIPS computation in the evaluation. The LPIPS computation is called as follows:
https://github.com/YuLiu-LY/SlotLifter/blob/0ea13606101430e6371608284c38f897934b961e/util/metrics.py#L244C13-L244C50

However, `preds` and `targets` have ranges [0,1], where as the lpips package (https://github.com/richzhang/PerceptualSimilarity) assumes inputs to be in the range [-1, 1]. 

I think the easiest fix would be to call `v(preds, targets, normalize=True).mean()`, see: https://github.com/richzhang/PerceptualSimilarity/blob/082bb24f84c091ea94de2867d34c4544f68e0963/lpips/lpips.py#L113

Would it be possible to recompute the evaluation metrics with this fix and share them? I would do this myself, but I actually can not reproduce the reported metrics exactly. For example on CLEVR-567, the reported metrics are:
<img width="1070" height="695" alt="Image" src="https://github.com/user-attachments/assets/88487265-5d6c-4a98-8a6a-08af46b079ff" />
However, when I run the evaluation with the provided checkpoint, I get:
```
        test/ari            0.9457603693008423
       test/ari_fg          0.9314202070236206
       test/lpips           0.02154792845249176
       test/nv_ari          0.8904937505722046
        test/psnr            35.57339859008789
        test/ssim           0.9648535847663879
```
So somehow I can reproduce the reported object segmentation results of `SlotLifter\dagger`, but not the novel view synthesis results. Were the results obtained from different training runs?

(I use the following command to run the evaluation: `python trainer/train.py cfg=uorf cfg.job_type=test cfg.exp_name=eval_clevr_567_seed1 cfg.group=clevr_567 cfg.subset=clevr_567 cfg.num_workers=8 cfg.chunk=16384 cfg.test_percent=1.0 cfg.ckpt_path=pretrained_weights/checkpoints/clevr_567.ckpt cfg.num_slots=8 cfg.seed=0 cfg.logger=wandb`)

It would be great if updated numbers (with the exact checkpoints that reproduce the NVS results from the paper) could be shared with the LPIPS fix for all the evaluated datasets. Alternatively it would be great if the checkpoints that were used for the reported NVS results could be shared. 

If I am mistaken and there is actually no bug in the LPIPS computation, please just let me know.
Thanks a lot and best,
Philipp

**Comment by YuLiu-LY** [comment]
Thank you for your interest in our work. We certainly didn't notice this during our evaluation. I suspect the input range of [0, 1] or [-1, 1] wouldn't have a significant impact. We've provided one checkpoint from multiple experiments. Our paper mentions that, to ensure fair comparison with previous papers, we report the best rendering metrics (including LPIPS) among all checkpoints. The checkpoints we provide have better segmentation metrics, while rendering metrics are generally slightly worse, but within the margin of error. You can evaluate our method using the checkpoint we provide.

**Comment by pschroeppel** [comment]
Hi,
thanks a lot for the reply and the clarifications. I will then re-evaluate SlotLifter with the provided checkpoint. The different input range does not have a huge impact, but there is some difference. 
Thanks a lot and best,
Philipp
